src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -294,7 +294,7 @@ void Engine::GetPerformanceStreams(Config& config, const std::shared_ptr<ngraph:
 
     get_num_streams(streams, ngraphFunc, config);
 
-    hints_props.insert({latency_name, std::to_string(latency_streams)});
+    hints_props.insert({latency_name, std::to_string(config.streamExecutorConfig._streams)});
```

Comment:
In what cases `config.streamExecutorConfig._streams` might be differ from `latency_streams`?

Reply:
@sunxiaoxia2022 could you give more details on why choose `config.streamExecutorConfig._streams` rather than `get_num_numa_nodes()`?